doing-ml.py

```diff
@@ -0,0 +1,9 @@
+import os
+
+def do_ml(hyperparam, params):
+    print("Starting")
+    print("Doing magic")
+    print("Done")
+
+
+do_ml()
```

Comment:
This is not working!

If you run this code the following happens:

```{bash}
$ python doing-ml.py 
Traceback (most recent call last):
  File "doing-ml.py", line 9, in <module>
    do_ml()
TypeError: do_ml() missing 2 required positional arguments: 'hyperparam' and 'params'
```

You need to pass something to the function (or using some by default arguments)